app/services/analytics_events.rb

```diff
@@ -572,7 +572,7 @@ def idv_verify_in_person_troubleshooting_option_clicked(flow_path:, **extra)
   # The user agrees to allow us to access their data via api call
   def idv_inherited_proofing_agreement_submitted(success:, errors:, flow_path:, step:, **extra)
     track_event(
-      'Idv: inherited proofing agreement submitted',
+      'IdV: inherited proofing agreement submitted',
```

Comment:
in general, I am not a fan of changing casing like this. These are string keys that get written to logs, and so changing this will break old queries

In this case, I believe these events have not been written at all in prod, so it's probably fine

Reply:
Agreed. These are newly defined events, that are breaking convention.

Reply:
@jscodefix Not a show stopper for me, but since we're breaking the current convention, perhaps we should go the whole way and also do away with the acronym altogether "Identity Verification". I've been doing away with the acronyms as I've been going along, since it's been one of the goals we've been trying to accomplish over time as I understand it. But, once this is in, we have to live with it, and it'd take a migration (after_party, whatever we use) to change it.

Reply:
This is not breaking convention; it is adhering to the established convention.